src/utils/utils.go

```diff
@@ -60,3 +60,11 @@ func isABuildFile(name string, config *core.Configuration) bool {
 	}
 	return false
 }
+
+// Max is an incredibly complicated function, which is presumably why it didn't make it into the Go stdlib
```

Comment:
There's one for floats but not for ints. The float implementation is non-trivial though which I guess is the argument there.